agr_api/src/main/java/org/alliancegenome/api/controller/DiseaseController.java

```diff
@@ -317,6 +317,18 @@ public JsonResultResponse<GeneDiseaseAnnotationDocument> getDiseaseAnnotationsBy
 		}
 	}
 
+	/**
+	 * replace Saccharomyces cerevisiae with Saccharomyces cerevisiae S288C
```

Comment:
This method just splits the input into a list? So this comment doesn't fit what the method does?